lib/pbench/server/database/models/users.py

```diff
@@ -27,36 +29,29 @@ class UserSqlError(UserError):
     key; the cause will specify the original SQLAlchemy exception.
     """
 
-    def __init__(self, operation: str, params: JSONOBJECT, cause: str):
-        self.operation = operation
-        self.params = params
+    def __init__(self, cause: Exception, **kwargs):
+        super().__init__(f"User SQL error: '{cause}'", kwargs)
+        self.kwargs = kwargs
         self.cause = cause
```

Comment:
It's probably not a big deal, but in the other modules I think you assigned `self.kwargs` _after_ assigning `self.cause`....

Reply:
I tried to be consistent, and I have no idea how this one slipped off its leash...